AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -918,13 +918,15 @@ - (CGSize)viewportSizeForRangeController:(ASRangeController *)rangeController
 - (ASInterfaceState)interfaceStateForRangeController:(ASRangeController *)rangeController
```

Comment:
This looks like exactly what we need.  In a future diff, let's factor this into the range controller itself, and do something like interfaceStateForDataSource:withWindow: and then deduplicate / share the logic.